src/components/Expensicons/Icons.js

```diff
@@ -0,0 +1,14 @@
+import ICON_NAMES from './ICON_NAMES';
+import ExpensifyCashLogoIcon from '../../../assets/images/expensify-cash.svg';
+import PinIcon from '../../../assets/images/pin.svg';
+
+export default {
+    [ICON_NAMES.EXPENSIFY_CASH_LOGO]: {
```

Comment:
I see this more of a branding asset than a logo. I personally would leave this out of here.

Ideally our icon system is used for UI icons like these:
![image](https://user-images.githubusercontent.com/2319350/103920316-fc80a200-50c5-11eb-9449-717dd8e70cfa.png)

So whatever you are doing in this PR, you should optimize for these one-color UI icons. Everything else is typically some kind of branding asset.